spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -115,6 +115,10 @@ enum class SpdxLicense(
     BITTORRENT_1_0("BitTorrent-1.0"),
     /** BitTorrent Open Source License v1.1 */
     BITTORRENT_1_1("BitTorrent-1.1"),
+    /** SQLite Blessing **/
+    BLESSING(blessing),
+    /** Blue Oak Model License 1.0.0 **/
+    BLUEOAK_1_0_0(BlueOak-1.0.0,
```

Comment:
There are some syntax errors here and above. I wonder how this could have happened, did you create these entries manually? Because if I run `./gradlew :spdx-utils:generateSpdxEnums` all is fine, also see my `regenerate-spdx-enums` branch. Also, my branch has a whole lot more license text files for some reason.

Reply:
So I propose to merge https://github.com/heremaps/oss-review-toolkit/pull/1798 instead.